lib/hammer_cli_foreman_ansible/ansible_inventories.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module HammerCLIForemanAnsible
+  class AnsibleInventoriesCommand < HammerCLIForeman::Command
+    resource :ansible_inventories
+
+    class HostsCommand < HammerCLIForeman::Command
+      action :hosts
+      command_name 'hosts'
+      option '--as-json', :flag, 'Full response as json'
+
+      def print_data(data)
+        return super unless option_as_json?
+
+        puts JSON.pretty_generate(data)
+      end
+      build_options
+      extend_with(HammerCLIForemanAnsible::CommandExtensions::Inventory.new)
+
```

Comment:
Please, don't leave such empty lines, I guess it's better if it would look like:

```ruby
class
  # ...
  build_options
  
  extend_with(HammerCLIForemanAnsible::CommandExtensions::Inventory.new)
end
```